src/Illuminate/Notifications/Channels/DatabaseChannel.php

```diff
@@ -55,7 +55,7 @@ protected function buildPayload($notifiable, Notification $notification)
     {
         return [
             'id' => $notification->id,
-            'type' => get_class($notification),
+            'type' => $notification->type_alias ?? get_class($notification),
```

Comment:
Should be camelCase?

```suggestion
            'type' => $notification->typeAlias ?? get_class($notification),
```